CMakeLists.txt

```diff
@@ -9,8 +9,40 @@ include_directories(bomberman lib/wal/sources)
 add_subdirectory(${PROJECT_SOURCE_DIR}/lib/wal)
 add_subdirectory(${PROJECT_SOURCE_DIR}/lib/Ray)
 
+set(SOURCES
```

Comment:
why aren't you using the normal technic
(add_executable wth all files)

Reply:
Because I do not want to copy paste all files.